components/navbar/chatlist.js

```diff
@@ -22,15 +22,15 @@ function ChatList({ currentUser, userID, friendsList, isDarkTheme }) {
 
   useEffect( () => {
     findMessagesPhotos()
-    setTime()
+    const setTime = () => {
```

Comment:
ahh Interesting. Declaring the func inside of the useEffect has a different result than just calling it within?